Source/Bogus/Extensions/Sweden/ExtensionsForSweden.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Linq;
+using Bogus.DataSets;
+
+namespace Bogus.Extensions.Sweden
+{
+   /// <summary>
+   /// API extensions specific for a geographical location.
+   /// </summary>
+   public static class ExtensionsForSweden
+   {
+      /// <summary>
+      /// Swedish national identity number (personummer)
+      /// </summary>
+      public static string Personummer(this Person person)
```

Comment:
Should the name of this extension method be `Personnummer` with two `nn`s?
https://en.wikipedia.org/wiki/Personal_identity_number_(Sweden)

Reply:
You are absolutely right! thanks i'll fix

Reply:
fixed